engine/platform/sdl/vid_sdl.c

```diff
@@ -708,7 +712,7 @@ static qboolean VID_CreateWindowWithSafeGL( const char *wndname, int xpos, int y
 	while( glw_state.safe >= SAFE_NO && glw_state.safe < SAFE_LAST )
 	{
 #if SDL_VERSION_ATLEAST( 2, 0, 0 )
-		host.hWnd = SDL_CreateWindow( wndname, xpos, ypos, w, h, flags );
+		host.hWnd = SDL_CreateWindow( wndname, SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, w, h, flags );
```

Comment:
You don't need to do this here because `xpos` and `ypos` are passed from `VID_CreateWindow`.

By default, it's SDL_WINDOWPOS_UNDEFINED, but tries to take the window position saved in the config.